cmd/web-handlers.go

```diff
@@ -2210,6 +2210,8 @@ func toWebAPIError(ctx context.Context, err error) APIError {
 		return getAPIError(ErrBadDigest)
 	case IncompleteBody:
 		return getAPIError(ErrIncompleteBody)
+	case PrefixAccessDenied:
```

Comment:
is this required?

Reply:
This is not required, no changes are needed in web-handlers.go AFAICS